docs/generated/available-plugins.md

```diff
@@ -7670,7 +7670,7 @@ Auto generated, please only modify https://github.com/fastlane/fastlane/blob/mas
 
 <p class="via-text">via William Lauze</p>
 
-> Fork for: Automated version managment and generator of release notes.
+> Fork for: Automated version management and generator of release notes.
```

Comment:
These references are pulled from 3rd party repositories. One repo made this typo, and 4 other repos forked it so the typo spread 😂 

Ref.: https://docs.fastlane.tools/plugins/available-plugins/

We'll have to revert these as this file gets autogenerated on every deploy 😅